sys/kern/tty.c

```diff
@@ -881,15 +881,20 @@ static int tty_get_termios(tty_t *tty, struct termios *t) {
 }
 
 static int tty_set_termios(tty_t *tty, u_long cmd, struct termios *t) {
-  int error;
+  int err;
 
   SCOPED_MTX_LOCK(&tty->t_lock);
   if (tty_detached(tty))
     return ENXIO;
 
+  /* Processes in background process groups are restricted from modifying
+   * their controlling terminal's settings. */
```

Comment:
What do you think about inserting the comment by `tty_check_background` definition instead of repeating it?

Reply:
Good idea. Done.